Makefile

```diff
@@ -8,19 +8,22 @@ $(WORKDIR)/go-libp2p-daemon:
 	mkdir -p bin/go-libp2p-daemon
 
 clone: $(WORKDIR)/go-libp2p-daemon
-	git clone https://github.com/libp2p/go-libp2p-daemon.git bin/go-libp2p-daemon && cd $(WORKDIR)/go-libp2p-daemon && git checkout $(COMMIT)
+	git clone https://github.com/libp2p/go-libp2p-daemon.git bin/go-libp2p-daemon; cd $(WORKDIR)/go-libp2p-daemon && git checkout $(COMMIT)
 
-$(TARGETS): clone
+darwin: clone
 	cd bin/go-libp2p-daemon/p2pd && GOOS=$@ go build -o p2pd-$@ && mv p2pd-$@ ../../
 
+linux: clone
+	cd bin/go-libp2p-daemon/p2pd && GOARCH=amd64 GOOS=$@ go build -o p2pd-$@ && mv p2pd-$@ ../../
```

Comment:
This means only amd64 is supported on linux, right?  Is there any way to have arm as well?  A bit niche but sometimes it's necessary to to run linux on a mac in a VM to debug things, which means it's running on arm.

Reply:
The interop test on js-libp2p CI runs on amd64 linux. It would be easy to add a build target in the Makefile, but currently different architectures are not supported in `download.js`.

Reply:
Not going to block on this though